modules/backend/widgets/Lists.php

```diff
@@ -581,6 +587,83 @@ public function prepareQuery()
         return $query;
     }
 
+    /**
+     * Check if the current sort column has a custom map.
+     * We check if the column's sortable property is an array or string.
+     *
+     * array: represents the order of which the values should be returned
+     * string: references a function on the relevant model that defines or handles the custom sort
+     *
+     * @return boolean
+     */
+    public function sortColumnHasCustomSorting(): bool
```

Comment:
@bradietilley We're not using return types yet until we do a big refactor of the OCMS code.
```suggestion
    public function sortColumnHasCustomSorting()
```